Rakefile

```diff
@@ -1,11 +1,6 @@
 require "bundler/gem_tasks"
-require "rake/testtask"
+require "rspec/core/rake_task"
 
-Rake::TestTask.new do |t|
-  t.libs.push "lib"
-  t.libs.push "test"
-  t.test_files = FileList["test/**/*_test.rb"]
-  t.verbose = true
-end
+RSpec::Core::RakeTask.new(:spec)
```

Comment:
if you want to maintain the behavior where we print ruby compiler warnings, this should be changed to:

```ruby
RSpec::Core::RakeTask.new(:spec) do |task|
  task.ruby_opts = "-W"
end
```